politeiawww/wsdcrdata.go

```diff
@@ -51,8 +90,20 @@ func (w *wsDcrdata) removeSub(event string) {
 	delete(w.subscriptions, event)
 }
 
+// removeAllSubs removes all of the subscriptions from the subscriptions map.
+func (w *wsDcrdata) removeAllSubs() {
+	w.Lock()
+	defer w.Unlock()
+
+	w.subscriptions = make(map[string]struct{})
+}
+
 // isSubscribed returns whether the client is subscribed to the provided event.
 func (w *wsDcrdata) isSubscribed(event string) bool {
+	if w.isShutdown() {
```

Comment:
You don't need to check this if the subscriptions are cleared out properly in the close() method.  You really will only need to check isShutdown before making external calls.  Like subscribe or ping.